src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,80 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
 public class WorkWithFile {
+    private static final String ADDED = "buy";
+    private static final String DECREASED = "supply";
+    private static final String CSV_SEPARATOR = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        HashMap<String, Integer> map = readingFromFile(fromFileName);
+        String result = formingResult(map);
+        String resultFromFile = writingFromFile(toFileName, result);
+    }
+
+    public HashMap<String, Integer> readingFromFile(String fromFileName) {
+        HashMap<String, Integer> sellAndBuy = new HashMap<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String lineWords = bufferedReader.readLine();
+            while (lineWords != null) {
+                String[] line = lineWords.split(CSV_SEPARATOR);
+                String key = line[0];
+                Integer value = Integer.parseInt(line[1]);
+                if (sellAndBuy.containsKey(key)) {
+                    sellAndBuy.put(key, sellAndBuy.get(key) + value);
+                } else {
+                    sellAndBuy.put(key, value);
+                }
+                lineWords = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file", e);
+        }
+        return sellAndBuy;
+    }
+
+    public String formingResult(HashMap<String, Integer> sellAndBuy) {
+        StringBuilder builder = new StringBuilder();
+        for (Map.Entry<String, Integer> kvp : sellAndBuy.entrySet()) {
```

Comment:
you don't need for each here, you have only 2 keys, don't run 2 loops to get them